XeroNetStandardApp/Utilities/TokenUtilities.cs

```diff
@@ -20,6 +24,23 @@ public struct State
       this.state = state;
     }
   }
+
+
+  public async static Task<XeroOAuth2Token> GetXeroOAuth2Token(XeroConfiguration xeroConfig)
```

Comment:
It will be good to use this static method across all controllers and set the existing GetStoredToken() to a private static method. 
Will do in a different PR to keep this PR focused on the InvocieSyncController